src/main/java/uk/gov/hmcts/reform/unspec/service/bankholidays/BankHolidaysApi.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.unspec.service.bankholidays;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@FeignClient(name = "bank-holidays-api", url = "${bankHolidays.api.url}")
+public interface BankHolidaysApi {
+
+    @GetMapping(path = "/bank-holidays.json")
+    BankHolidays retrieveAll();
+}
```

Comment:
thoughts on removing this from unspec and placing in it's own calendar / bank holidays folder?

Reply:
Probably moving this to a submodule would be another option. Will do that afterwards as there are branches out of this branch.